platform.go

```diff
@@ -40,64 +70,115 @@ var (
 		{"windows", "amd64", true},
 	}
 
-	Platforms_1_1 = append(Platforms_1_0, []Platform{
+	Platforms_1_1 = addDrop(Platforms_1_0, []Platform{
 		{"freebsd", "arm", true},
 		{"netbsd", "386", true},
 		{"netbsd", "amd64", true},
 		{"netbsd", "arm", true},
 		{"plan9", "386", false},
-	}...)
+	}, nil)
 
-	Platforms_1_3 = append(Platforms_1_1, []Platform{
+	Platforms_1_3 = addDrop(Platforms_1_1, []Platform{
 		{"dragonfly", "386", false},
 		{"dragonfly", "amd64", false},
 		{"nacl", "amd64", false},
 		{"nacl", "amd64p32", false},
 		{"nacl", "arm", false},
 		{"solaris", "amd64", false},
-	}...)
+	}, nil)
 
-	Platforms_1_4 = append(Platforms_1_3, []Platform{
+	Platforms_1_4 = addDrop(Platforms_1_3, []Platform{
 		{"android", "arm", false},
 		{"plan9", "amd64", false},
-	}...)
+	}, nil)
 
-	Platforms_1_5 = append(Platforms_1_4, []Platform{
+	Platforms_1_5 = addDrop(Platforms_1_4, []Platform{
```

Comment:
I noticed some very strange things happening if multiple `appends` were done around here, with the `1.6` and `1.7` slices being all messed up, so I changed everything to use `addDrop`, which makes full copies.